test/metabase/api/database_test.clj

```diff
@@ -563,6 +563,15 @@
 
 (defmethod driver/database-supports? [::no-nested-query-support :nested-queries] [_driver _feature _db] false)
 
+(defn- get-all
+  [endpoint existing-ids]
+  (let [new?        (complement (set existing-ids))
+        dbs         (->> (mt/user-http-request :rasta :get 200 endpoint)
+                         :data
+                         (filter (comp new? :id)))]
+    {:data  dbs
+     :total (count dbs)}))
```

Comment:
my quest to make it so that I can run tests locally with a polluted app DB continues…